src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.ts

```diff
@@ -90,98 +85,54 @@ export class WorkItemLinkComponent implements OnInit, OnChanges {
         
     }
 
-    onSelectRelation(relation: LinkType): void{
+    onSelectRelation(relation: any): void{
```

Comment:
Actually it's an object and I don't have any type defined for that kind of object. I'll add one then. Let me see.